pysankey/sankey.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-"""
+r"""
 Produces simple Sankey Diagrams with matplotlib.
 @author: Anneya Golob & marcomanz & pierre-sassoulas & jorwoods
```

Comment:
@vgalisson you should add your name here :)

Reply:
done